eclipse-theia/theia
# URI:folders with '%20' in their name can be opened

The commit adds the encoding of URIs. After this update, we can now
open workspaces with reserved symbols like '%' in their name.





#### What it does

FIxes #8909 


#### How to test

1. Try to open a workspace with '%' in the name
2. Check if it opens correctly
#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
